.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+<!--
+Thanks for sending a pull request!
+-->
+
+#### What this PR does / why we need it:
+
+#### Which issue(s) this PR fixes:
+<!--
+Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
+-->
+Fixes #
+
+#### Special notes for your reviewer:
+<!--
+Any additional special notes for your reviewer.
+-->
+
+## Steps to reproduce
```

Comment:
Always found it odd that this header was different from the others in KOTS. Can we make them consistent?
```suggestion
#### Steps to reproduce
```

Reply:
Yes, thank you I've taken in the suggestion